Empire/Empire.cpp

```diff
@@ -1118,7 +1118,7 @@ void ProductionQueue::Update() {
             std::tie(item_cost, build_turns) = queue_item_costs_and_times[key];
             float total_item_cost = item_cost * element.blocksize;
 
-            float additional_pp_to_complete_element = total_item_cost * (1.0f - element.progress); // additional PP, beyond already-accumulated PP to finish this element once
+            float additional_pp_to_complete_element = total_item_cost * (element.remaining - element.progress); // additional PP, beyond already-accumulated PP to finish this element once
```

Comment:
Can you add some comments before this line explaining why using `element.remaining` here makes sense? I think it does make sense, but conceptually the `remaining` integer count doesn't immediately seem to be in the same units as the float `progress`, so a note about how `progress` ranges from 0 to 1 and thus can be usefully subtracted from `remaining` would be helpful.